auslib/db.py

```diff
@@ -185,7 +185,7 @@ class AUSTable(object):
     """
 
     def __init__(self, db, dialect, history=True, versioned=True, scheduled_changes=False,
-                 onInsert=None, onUpdate=None, onDelete=None):
+                 scheduled_changes_kwargs={}, onInsert=None, onUpdate=None, onDelete=None):
```

Comment:
The current code looks alright from a global var point of view (even though like you said, `scheduled_changes_kwargs` will remain shared).

As far as I know, a global var happens with this kind of scenario:
```py
def my_func(foo={}):    # Side note: meh, it's not a pure function
    foo['bar'] = 'baz'
    return foo
 
a = my_func()
a['baz'] = 'real baz'

b = my_func()    # b == {'baz': 'real baz', 'bar': 'baz'}
```